File: Model/cus_viewreply.py
#!c:\python\python
from lib import DBConnection as c
import cgi, cgitb,json,os
print("Content-type: text/html\n\n")
cgitb.enable()
form = cgi.FieldStorage()
action=form.getvalue("action")
if action == "getreply":
    #data = json.loads(form.getvalue("data"))
    v=form.getvalue("id")
    #sql = "SELECT customer.cust_id,customer.name,question.question_desc,question.reply_desc FROM `customer`INNER JOIN question ON customer.cust_id = question.customer_id "
    sql = "SELECT customer.cust_id,customer.name,question.question_desc,question.reply_desc FROM `customer`INNER JOIN question ON customer.cust_id = question.customer_id where question.customer_id='%s'" %v
    res = c.db.getJSON(sql)
    print(res)

File: Controller/cus_viewreply.js
$(function () {
    Getreply()
})
function Getreply() {
    user=localStorage.getItem("uid")
    data = { "action": "getreply","id":user }
    //alert(user)
    //alert(data)
    console.log(data)
    $.post("Model/cus_viewreply.py", data, function (res) {
        res = JSON.parse(res)
        console.log(res)
        //alert(res)
        if (res != null) {
            $("#tb").show()
            $.each(res, function (k, v) {
                s = '<tr><td>' + v.name+ '</td>' +
                    '<td>' + v.question_desc + '</td>' +
                    '<td>' + v.reply_desc + '</td>' 
                    
                    
                $("#stds").append(s)
            })
        } else {
            
         
        }
    })
}